native/src/main/java/io/ballerina/stdlib/sql/parameterprocessor/DefaultResultParameterProcessor.java

```diff
@@ -854,6 +854,13 @@ public Object processCustomTypeFromResultSet(ResultSet resultSet, int columnInde
         throw new ApplicationError("Unsupported SQL type " + columnDefinition.getSqlName());
     }
 
+    @Override
+    public Object getAndConvertXml(ResultSet resultSet, int columnIndex, int sqlType, Type balType)
```

Comment:
With the changes I suggested, there should be no changes to module-specific ResultParameterProcessor.

Reply:
updated in https://github.com/ballerina-platform/module-ballerina-sql/pull/329/commits/ec7082e053a0528f35ea6b01e858df0b0ba0fb6f